### PR TITLE
Add profile DISC

### PR DESCRIPTION
**Pull request by jennyliang220**

### CODE REVIEW

File: lib/rules.json [hunk 1]
```diff
@@ -1982,6 +1982,113 @@
...
+                                "<a href=\"https://www.w3.org/2021/Process-20211102/#recs-and-notes\">Recommendation track</a>",
+                                "&lt;a href=\"https://www.w3.org/2021/Process-20211102/#recs-and-notes\"&gt;Recommendation track&lt;/a&gt;"
+                            ],
+                            "implEstimation": "It <span class=\"rfc2119\">should</span> include an estimated date by which time the Working Group expects to have sufficient implementation experience.",
+                            "featAtRisk": "It <span class=\"rfc2119\">must</span> identify any \"features at risk\" declared by the Working Group (as defined in section <a href=\"https://www.w3.org/Consortium/Process/#candidate-rec\">6.4 of the W3C Process Document</a>).",
+                            "customParagraph": true,
+                            "changesList": [
+                                "must",
+                                ""
+                            ],
```
Comment (deniak) [inline lib/rules.json:2062]: These 4 rules don't apply to Discontinued Drafts

```suggestion
```

**Review by deniak — CHANGES_REQUESTED** [review]

File: lib/rules.json [hunk 2]
```diff
@@ -1982,6 +1982,113 @@
...
+                             "stability": "It <span class=\"rfc2119\">must</span> set expectations about the (in)stability of the document. The <span class=\"rfc2119\">recommended</span> text is: <blockquote class=\"boilerplate\"> <p>Publication as a Discontinued Draft imply that this document no longer intended to advance or to be maintained. It is inappropriate to cite this document as other than abandoned work.</p> </blockquote> <div class=\"source\"> <span style=\"font-style: italic\">Include this source code</span>:<br><code>&lt;p&gt;Publication as a Discontinued Draft imply that this document no longer intended to advance or to be maintained. It is inappropriate to cite this document as other than abandoned work.&lt;/p&gt;</code></div>",
```
Comment (deniak) [inline lib/rules.json:2063]: template was updated after discussions with comm team
```suggestion
                             "stability": "It <span class=\"rfc2119\">must</span> set expectations about the (in)stability of the document. The <span class=\"rfc2119\">recommended</span> text is: <blockquote class=\"boilerplate\"> <p>Publication as a Discontinued Draft implies that this document is no longer intended to advance or to be maintained. It is inappropriate to cite this document as other than abandoned work.</p> </blockquote> <div class=\"source\"> <span style=\"font-style: italic\">Include this source code</span>:<br><code>&lt;p&gt;Publication as a Discontinued Draft imply that this document no longer intended to advance or to be maintained. It is inappropriate to cite this document as other than abandoned work.&lt;/p&gt;</code></div>",
```

File: lib/rules/sotd/stability.js [hunk 3]
```diff
@@ -15,6 +15,9 @@ function findSW(candidates, sr) {
     } else if (sr.config.longStatus === 'Statement') {
         wanted =
             'A W3C Statement is a specification that, after extensive consensus-building, has received the endorsement of the W3C and its Members.';
+    } else if (sr.config.longStatus === 'Discontinued Draft') {
+        wanted =
+            'Publication as a Discontinued Draft imply that this document no longer intended to advance or to be maintained. It is inappropriate to cite this document as other than abandoned work.';
```
Comment (deniak) [inline lib/rules/sotd/stability.js:20]:
```suggestion
            'Publication as a Discontinued Draft implies that this document is no longer intended to advance or to be maintained. It is inappropriate to cite this document as other than abandoned work.';
```

**Review by deniak — APPROVED** [review]

File: lib/profiles/TR/DISC.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    status: 'DISC',
+    longStatus: 'Discontinued Draft',
+    styleSheet: 'W3C-DISC',
+    stabilityWarning: true,
+    track: 'Recommendation',
+};
+
+const base = require('../base');
+let rules = base.insertAfter(require('../TR').rules, 'sotd.supersedable', [
+    require('../../rules/structure/security-privacy'),
```
Comment (deniak) [inline lib/profiles/TR/DISC.js:12]: why add that rule for a discontinued draft?

**Review by deniak — CHANGES_REQUESTED** [review]

File: lib/profiles/TR/DISC.js (hunk 4 above)
Comment (jennyliang220) [reply lib/profiles/TR/DISC.js:12]: There's **Privacy and Security** section in https://w3c.github.io/tr-design/p2021mockup/disc-6.3.12.1.html#sec-privacy-security

Comment (deniak) [reply lib/profiles/TR/DISC.js:12]: Pubrules only checks the SOTD and the headers of the documents. The template is just using an existing document that happens to have a security and privacy section but it shouldn't be required for discontinued draft

**Review by deniak — APPROVED** [review]

**Pull request merged**